### ISSUE TITLE
inconsistent behaviour of dialogue manager

### ISSUE THREAD

**Issue by riccardopinosio** [rhasspy/rhasspy#186]
Hi,

I'm hoping you can help me debug some inconsistent behaviour of the DialogueManager.
I have rhasspy running on a pi4 on docker. All services are activated with the recommended settings other than intent handling, which is disabled. Rhasspy has been freshly restarted.

Then I use MQTT explorer to publish this message on hermes/dialogueManager/startSession:

```
{
  "init": {
    "type": "action",
    "canBeEnqueued": 1,
    "text": "hi"
  },
  "siteId": "satellite-study"
}
```
Now, this works as expected: the speaker says "hi" and I can then issue a command, whose intent is recognized (it appears at hermes/nlu/intentParsed).
After a while, the session is terminated: a message is published on hermes/dialogueManager/sessionEnded with "termination":{"reason":"timeout"}. Fine.
However, after this if I try to initiate another action session, it looks like the session is enqueued and is never started. Only if I specify "canBeEnqueued":0 in the message is the action session started.

It seems to me that this should not happen, i.e., as soon as the session is ended because of timeout I should be able to start a new one without it being put in the queue. A Cursory look at the code suggests this to be the case.
Any help on how I can debug this further is appreciated.

**Comment by Chreece** [comment]
I posted an issue? which I think is related to this.
I don't see any sessionEnded after intend recognition and event sending to Home Assistant, only after a time out.

**Comment by synesthesiam** [comment]
Thanks for the report! This bug will be fixed in 2.5.10 :+1:

**Comment by synesthesiam** [comment]
Should be fixed in latest 2.5.10.
